packages/vscode-dependencies-validation/src/editorChanges.ts

```diff
@@ -0,0 +1,63 @@
+import type {
+  DiagnosticCollection,
+  TextDocument,
+  TextDocumentChangeEvent,
+  TextEditor,
+} from "vscode";
+import { refreshDiagnostics } from "./diagnostics";
+import { clearDiagnostics } from "./util";
+import { VscodePackageJsonChangesConfig } from "./vscodeTypes";
+
+export function subscribeToPackageJsonChanges(
+  vscodeConfig: VscodePackageJsonChangesConfig
+): void {
+  const { window, subscriptions, workspace, diagnosticCollection } =
+    vscodeConfig;
+
+  if (window.activeTextEditor) {
```

Comment:
can you link to the template you used for this method?

Reply:
I mean in a comment

Reply:
https://github.com/microsoft/vscode-extension-samples/blob/e958fb7d45abda8177f413062318447ce59c780f/code-actions-sample/src/diagnostics.ts#L47